spring-kafka-docs/src/main/antora/modules/ROOT/pages/kafka/receiving-messages/class-level-kafkalistener.adoc

```diff
@@ -62,3 +62,25 @@ void listen(Object in, @Header(KafkaHeaders.RECORD_METADATA) ConsumerRecordMetad
     ...
 }
 ----
+
+Also, this won't work as well.
+The `payload` value will be injected as the `topic`.
+
+[source, java]
+----
+@KafkaHandler(isDefault = true)
+public void listenDefault(String payload, @Header(KafkaHeaders.RECEIVED_TOPIC) String topic) {
+    // payload.equals(topic) is True.
+    ...
+}
+----
+
+If you need discrete custom headers in a default method, use this:
```

Comment:
I thought we discussed before that no "you", "me", "we" or even "they" in the technical documentation.